engine/access/rest/http/request/get_transaction.go

```diff
@@ -90,6 +141,55 @@ func (g *GetTransactionResult) Build(r *common.Request) error {
 	return err
 }
 
+// GetTransactionResultsByBlockID represents a request to get transaction results by
+// block ID or height, and contains the parsed and validated input parameters.
+type GetTransactionResultsByBlockID struct {
```

Comment:
since the endpoints support ID and height

```suggestion
type GetTransactionResultsByBlock struct {
```